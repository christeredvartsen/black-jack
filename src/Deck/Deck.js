import React from 'react';
import types from '../propTypes';

const Deck = ({ cards }) => (
  <p>There are {cards.length} cards left in the deck.</p>
);
Deck.propTypes = {
  cards: types.cards.isRequired,
};

export default Deck;