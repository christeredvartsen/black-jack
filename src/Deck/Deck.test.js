import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Deck from './Deck';

describe('<Deck />', () => {
  test('matches snapshot', () => {
    const cards = [
      { suit: 'HEARTS', value: '10' },
      { suit: 'SPADES', value: '8' },
      { suit: 'HEARTS', value: '9' },
    ];
    const tree = shallow(<Deck cards={cards} />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});