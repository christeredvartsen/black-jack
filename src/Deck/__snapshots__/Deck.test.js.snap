// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Deck /> matches snapshot 1`] = `
<p>
  There are 
  3
   cards left in the deck.
</p>
`;
